features/profile/constants.ts

```diff
@@ -44,87 +38,115 @@ export const referenceBadgeLabel = (t: TFunction) => ({
   ),
 });
 
+export enum SmokingLocation {
+    YES = 'yes',
+    OUTSIDE = 'outside',
+    NO = 'no',
+    WINDOW = 'window',
+    UNSPECIFIED = 'unspecified'
+}
```

Comment:
It's a bit unfortunate that the generated TS code from the old gRPC code uses enum for this... which I'd prefer us to steer away if possible as they spit out more code than what could be written as a simple object with a `as const` (const assertion) to achieve the same thing. This guy explains it better than I can without turning into a long comment (and in particular, the share link starts at the bit that arguably makes enums annoying to type/work with): https://youtu.be/jjMbPt_H3RQ?t=181